.drone.yml

```diff
@@ -1943,48 +1943,58 @@ clone:
 
 steps:
   - name: Set up exec runner storage
+    environment:
+      WORKSPACE_DIR: /tmp/build-darwin-amd64
     commands:
-      - mkdir -p /tmp/build-darwin-amd64
-      - chmod -R u+rw /tmp/build-darwin-amd64
-      - rm -rf /tmp/build-darwin-amd64/go
+      - mkdir -p $WORKSPACE_DIR
+      - chmod -R u+rw $WORKSPACE_DIR
+      - rm -rf $WORKSPACE_DIR/go $WORKSPACE_DIR/.ssh
 
   - name: Check out code
     environment:
       GITHUB_PRIVATE_KEY:
         from_secret: GITHUB_PRIVATE_KEY
+      WORKSPACE_DIR: /tmp/build-darwin-amd64
     commands:
```

Comment:
if drone doesn't set any bash options automatically, call `set -u` as the first command to protect against `$WORKSPACE_DIR` (and other var) typos

Reply:
It doesn't look like it does set options, but it's pretty hard to tell. I'll add the flag.